code/game/gamemodes/wizard/artefact.dm

```diff
@@ -82,6 +82,55 @@
 	rend_desc = "Gently wafting with the sounds of endless laughter."
 	icon_state = "clownrender"
 
+
+//--- BECAUSE IT'S FUCKING HILARIOUS --//
+/obj/item/weapon/veilrender/uncharged
+	name = "wicked blade"
+	desc = "A replica of a notorious eldritch artifact of immense power. Legend tells that those who capture two powerful souls with it will weild the weapon of legends."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "render"
+	item_state = "render"
+	force = 5 //shitty wizard build quality
+	throwforce = 0
+	w_class = 3
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	charges = 1
+	spawn_type = /obj/singularity/narsie/wizard
+	spawn_amt = 1
+	activate_descriptor = "the blood geometer's prison"
+	rend_desc = "You hear the sound of laughing slaughter demons."

```

Comment:
MUH LORE

NAR'SIE DOES NOT LIVE IN THE SAME DIMENSION AS SLAUGHTER DEMONS, ONE LIVES OUTSIDE SPACE AND THE OTHER JUST LIVES IN HELL